### PR TITLE
upgrade Makefile to use pelican parameters

### PR DESCRIPTION
**Pull request by alexsavio**
I was having issues running pelican through the old Makefile and I followed the instructions for the pelican that we are using. This is our Makefile with improvements from the newer version of pelican.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -65,8 +80,10 @@ stopserver:
 publish:
 	$(PELICAN) $(INPUTDIR) -o $(OUTPUTDIR) -s $(PUBLISHCONF) $(PELICANOPTS)
 
-rsync: publish
-	rsync -e "ssh -p $(SSH_PORT)" -P -auvz --delete --exclude="2013" --exclude="2014" --exclude="2015" --exclude="2016" --exclude="2016_static_20160429" $(OUTPUTDIR)/ $(SSH_USER)@$(SSH_HOST):$(SSH_TARGET_DIR)
+ssh_upload: publish
+	scp -P $(SSH_PORT) -r $(OUTPUTDIR)/* $(SSH_USER)@$(SSH_HOST):$(SSH_TARGET_DIR)
```
Comment (datapythonista) [inline Makefile]: @pdebuyl can you have a look at this

Comment (pdebuyl) [reply Makefile]: not merging this, we need rsync

**Comment by pdebuyl** [comment]
will have a closer look later

File: Makefile (hunk 1 above)
Comment (alexsavio) [reply Makefile]: Well, they renamed it to `rsync_upload`. I could rename it?

**Comment by pdebuyl** [comment]
published

**Comment by datapythonista** [comment]
@alexsavio is this PR still relevant.

To me, I can serve locally the website with `./develop_server.sh start` and publish it with `make sync`, I guess it's simple enough, not sure if the `rsync_upload` was solving a problem or trying to make things more efficient.

Btw @pdebuyl I think Alex doesn't have a user in the server, would be helpful if he can also deploy.

**Pull request closed without merging**